src/MSBuild.Sdk.SqlProj/MSBuild.Sdk.SqlProj.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿
-<Project Sdk="Microsoft.Build.NoTargets/3.7.0">
+<Project Sdk="Microsoft.Build.NoTargets/3.7.56">
```

Comment:
This seems unrelated. Was it necessary to change this?

Reply:
No. I just update this project, as I got build warnings in VS 17.12 about .netstandard 1.1. Should I revert?

Reply:
Reverted